xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -527,7 +527,7 @@ namespace PVR
 
     std::string DeleteTimer::GetLabel(const CFileItem &item) const
     {
-      if (item.GetPVRTimerInfoTag())
+      if (!item.GetPVRTimerInfoTag())
```

Comment:
This change is wrong. If the timer is a timer info tag, means it is displayed in timer/timer rules window, the context menu item shall be just "Delete", because it is obvious that the selected item is a timer. If selected item is not a timer info tag, e.g. an epg event in the guide window, then we name the context menu item "Delete timer" to make sure what will be deleted (not the epg event in my example).

Reply:
Yes, I agree that's wrong. Although I still can't see the reason for having two context menu descriptions for the same thing but worded differently. In the timers list you finish up with two entries "Delete Timer Rule" and just "Delete" rather than "Delete Timer Rule" and "Delete Timer" which to my mind makes more sense.

Reply:
> I still can't see the reason for having two context menu descriptions for the same thing but worded differently

 The reason is **context**. If context is timers/timer rules, then there is no need to put "timer" in the label, because this is redundant information. This is common practice for UI design guidelines in the applications I know.

Reply:
Yes, fine, but when you open the programme information from the context menu which shows just "Delete" we now get "Delete Timer":

![image2](https://user-images.githubusercontent.com/25765532/53699042-0eaf9c00-3ddc-11e9-82b5-161257f54e3e.png)

Reply:
Fair enough.